packages/lesswrong/components/recommendations/RecommendationsAndCurated.tsx

```diff
@@ -111,28 +119,23 @@ class RecommendationsAndCurated extends PureComponent<RecommendationsAndCuratedP
     // Disabled during 2018 Review [and coronavirus]
     const recommendationsTooltip = <div>
       <div>
-        Recently curated posts, as well as a random sampling of top-rated posts of all time
+        {forumTypeSetting.get() === 'EAForum' ?
```

Comment:
Not sure if that's LW style but I prefer the ternary operator to start on the next line

Reply:
Prevailing style seems to be this way.